tests/shared_test_functions.sh

```diff
@@ -269,8 +269,8 @@ setup_check_variables() {
 get_val_basename() {
 	val_basename=$1
```

Comment:
Looks like `val_basename` is not being used now?

Reply:
Huh.  I'll revisit this later.